v2/spanner-common/terraform/samples/configure-shared-vpc/main.tf

```diff
@@ -0,0 +1,38 @@
+# A service project gains access to network resources provided by its
+# associated host project.
+resource "google_compute_shared_vpc_host_project" "host" {
```

Comment:
Arent there templates which are published which already do this ? I don't really think these guides belong in the migration templates. They are sort of general purpose - how to setup a VPC sort of thing.

Reply:
> Arent there templates which are published which already do this ? 

This template does additional things which are specific to migration templates, for example - 
1. Add permissions for datastream to create a private connectivity link.
2. Add permissions for Dataflow to launch workers in the shared VPCs.

Hence, these templates are not generic, but specific to migrations. 

> They are sort of general purpose - how to setup a VPC sort of thing.

This one does not setup a VPC, it configures an existing VPC for a migration. Consider a scenario like our ongoing large customer. They would have needed to do all of the setup that we are doing in this template to *make their environment ready for a migration* - this template automates that process.